src/backend/app.ts

```diff
@@ -20,6 +21,7 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, '../../public')));
+app.use('/uploads', express.static(config.get('uploads')));
```

Comment:
is it breaks the backward capability for the previously uploaded images?

Reply:
nope